managedbeans/1.0/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta Managed Beans Spec 1.0"
+date: 2019-07-30
+summary: "Initial release of Managed Beans 1.0 for Jakarta EE 8"
+---
+Jakarta Managed Beans defines a set of basic services for container-managed objects
+with minimal requirements, otherwise known under the acronym “POJOs” (Plain Old Java Objects).
```

Comment:
“POJOs”  => "POJOs"  (ascii 34 quote char)

Reply:
Wow!  how did you find that?  :-)  Will clean up.